AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Utils/IdentifyTextResultTransformers.swift

```diff
@@ -53,79 +53,52 @@ class IdentifyTextResultTransformers: IdentifyResultTransformers {
     }
 
     static func processText(_ textractTextBlocks: [AWSTextractBlock]) -> IdentifyDocumentTextResult {
-
+        var fullText = ""
         var words = [IdentifiedWord]()
         var lines = [String]()
         var linesDetailed = [IdentifiedLine]()
         var selections = [Selection]()
-        var fullText = ""
         var tables = [Table]()
         var keyValues = [BoundedKeyValue]()
-        var blockMap = [String: AWSTextractBlock]()
         var tableBlocks = [AWSTextractBlock]()
         var keyValueBlocks = [AWSTextractBlock]()
+        var blockMap = [String: AWSTextractBlock]()
 
         for block in textractTextBlocks {
-            guard let text = block.text, let identifier = block.identifier else {
+            guard let identifier = block.identifier else {
                 continue
             }
 
-            guard let boundingBox = processBoundingBox(block.geometry?.boundingBox) else {
-                continue
-            }
-
-            guard let polygon = processPolygon(block.geometry?.polygon) else {
-                continue
-            }
-            let word = IdentifiedWord(text: text,
-                                      boundingBox: boundingBox,
-                                      polygon: polygon,
-                                      page: Int(truncating: block.page ?? 0))
-
-            let line = IdentifiedLine(text: text,
-                                      boundingBox: boundingBox,
-                                      polygon: polygon,
-                                      page: Int(truncating: block.page ?? 0))
+            blockMap[identifier] = block
```

Comment:
My mistake here.
I think line 72 should be kept, and line 84, 89 should be removed. 
Reference for how Android does: https://github.com/aws-amplify/amplify-android/blob/71aa76d145812170d2f9d8e3110ef907427e08ef/aws-predictions/src/main/java/com/amplifyframework/predictions/aws/service/AWSTextractService.java#L148
And it makes sense, we don't know what document our customer want to identify, so to put every block into `blockMap` in case need to fetch out in later processing